### PR TITLE
[#30] Add Travis

### PR DESCRIPTION
**Pull request by vitkarpov**
Travis runs `npm run-script test`. Now it runs only codestyle, in the future it will be something like

```
npm run-script codestyle && gulp tests
```

### CODE REVIEW

**Comment by vitkarpov** [comment]
@iamakulov

**Comment by vitkarpov** [comment]
image

The green tick looks pretty ;)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+language: node_js
+before_script: npm run-script build
+node_js:
+  - 0.10
```
Comment (iamakulov) [inline .travis.yml]: Let’s also add `4` (as a LTS version) and `latest`.

File: package.json [hunk 2]
```diff
@@ -10,6 +10,8 @@
   "license": "MIT",
   "main": "source/index.js",
   "scripts": {
+    "build": "./node_modules/.bin/gulp build",
```
Comment (iamakulov) [inline package.json]: This could be just `"build": "gulp build"`, as Node adds `./node_modules/.bin/` to `PATH` on its own.

Comment (iamakulov) [inline package.json]: This is also related to the `codestyle` line which I didn’t notice in the PR so feel free to add additional commit that updates the line if you’d like to.

**Comment by iamakulov** [comment]
@vitkarpov Great! Added a few notes.

File: package.json (hunk 2 above)
Comment (vitkarpov) [inline package.json]: Yep. Already noticed that in travis logs. Get rid of `./node_modules/.bin`

File: .travis.yml (hunk 1 above)
Comment (vitkarpov) [inline .travis.yml]: @iamakulov maybe it should be just `node` and that's it?

> Specifying node will run using the latest stable Node.js release

We don't really need to test code in nodejs, it's used only for build and tests. So seems it doesn't matter, right?

Comment (iamakulov) [inline .travis.yml]: Yep, you’re right, let’s leave just `node_js`.

**Comment by vitkarpov** [comment]
Done

**Comment by iamakulov** [comment]
:+1:

**Pull request merged**